### PR TITLE
Added script for Codechef Scrapper

### PR DESCRIPTION
**Pull request by smriti26raina**
# Description
A python script that scrapes the content of Codechef Problems, as per the user's input, and stores the problem information separately in different PDFs.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project.
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ ] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.
- Screenshot from 2021-04-06 01-30-31
- 
Screenshot from 2021-04-06 02-06-07

### CODE REVIEW

File: Codechef Scrapper/codechef.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    pdf.add_page()
+    pdf.set_font("Arial", size = 15)
+    pdf.cell(200, 10, txt = problem["title"], ln = 1, align = 'C')
+    pdf.multi_cell(200, 10, txt =problem["statement"], align = 'L')
+    pdf.multi_cell(200, 10, txt =problem["test_case"], align = 'L')
+    pdf.output(problem["title"]+".pdf")   
+    
+    
+#main function
+def main():
```
Comment (kaustubhgupta) [inline Codechef Scrapper/codechef.py:81]: put this in if __name__:

https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do/

File: Codechef Scrapper/readme.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Codechef Scraper
```
Comment (kaustubhgupta) [inline Codechef Scrapper/readme.txt:1]:
```suggestion
# Codechef Scraper
```

File: Codechef Scrapper/readme.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+This python script will let the user to scrape 'n' number of codechef problems from any category/difficulty in https://www.codechef.com/ ,as provided by the user. The functionality of the script is to gain the information regarding particular codechef problem in different PDFs.
+
+
+Prerequisite Steps:
```
Comment (kaustubhgupta) [inline Codechef Scrapper/readme.txt:5]:
```suggestion
## Prerequisite Steps:
```

File: Codechef Scrapper/readme.txt [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Download the required packages from the following command in you terminal.(Make sure you're in the same project directory) 
+pip3 install -r requirements.txt
+
+To run this script,you need to have selenium installed and configure webdriver to use chrome browser.You can directly download chrome driver from the link below-
+https://chromedriver.chromium.org/downloads
+Further,you can set the path to chromedriver using
+driver= webdriver.chrome("/usr/lib/chromium-browser/chromedriver")
+
+Running the script:
```
Comment (kaustubhgupta) [inline Codechef Scrapper/readme.txt:14]:
```suggestion
## Running the script:
```

File: Codechef Scrapper/readme.txt [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+After installing all the requirements,run this command in your terminal.
+python3 codechef.py
+
+Output:
```
Comment (kaustubhgupta) [inline Codechef Scrapper/readme.txt:18]:
```suggestion
## Output:
```

File: Codechef Scrapper/readme.txt [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+This script will generate 'n' number of different PDFs in a folder to store the problem information (problem title,problem statement,test cases) separately.
+
+
+Author:
```
Comment (kaustubhgupta) [inline Codechef Scrapper/readme.txt:22]:
```suggestion
## Author:
```

File: Codechef Scrapper/readme.txt [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+Smriti Raina [https://github.com/smriti26raina]
```
Comment (kaustubhgupta) [inline Codechef Scrapper/readme.txt:23]:
```suggestion
[Smriti Raina](https://github.com/smriti26raina)
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Rename the readme.txt to README.md and do the following changes:

File: Codechef Scrapper/readme.txt (hunk 7 above)
Comment (smriti26raina) [reply Codechef Scrapper/readme.txt:23]: How can I attach my output images to README.md?

**Comment by smriti26raina** [comment]
@kaustubhgupta Sir, I've made all the required changes. Please check it.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
The script works perfectly. I have a suggestion: Can you include the problem statement link at the end of every problem in PDF. It will add more value to the script. I know it was not mentioned in the description but I just realized that this was missing. If you can add it then go ahead if not then also mention the same. 

Do these changes in the readme:

**Comment by smriti26raina** [comment]
@kaustubhgupta, I've updated the script, adding the URL of every problem statement at the end of the PDF.

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM

**Pull request merged**